vendor/wp-now/src/symlink-manager.ts

```diff
@@ -0,0 +1,220 @@
+import { PHP, UnmountFunction } from '@php-wasm/universal';
+import fs from 'node:fs/promises';
+import { FileChangeInfo } from 'node:fs/promises';
+import path from 'path';
+import { createNodeFsMountHandler } from '@php-wasm/node';
+
+type mountedTarget = {
```

Comment:
Nitpick: Usually, type names start capitalized.
```suggestion
type MountedTarget = {
```